t/31-client_archive.t

```diff
@@ -36,7 +36,7 @@ subtest 'OpenQA::Client:Archive tests' => sub {
         );
         my $command = $t->ua->archive->run(\%options);
     }
-    'Archive functionality works as expected would perform correctly';
+    'Archive functionality works as expected would perform correctly' or BAIL_OUT "$@";
```

Comment:
In this case the original code did not `die` or `BAIL_OUT`, so I suggest to leave that.